scripts/run_tests.sh

```diff
@@ -29,5 +29,8 @@ fi
 flake8 .
 display_result $? 1 "Code style check"
 
+isort --check-only ./app ./tests
+display_result $? 2 "Import order check"
+
 py.test --maxfail=10 tests/ --strict
 display_result $? 2 "Unit tests"
```

Comment:
Should this be `3` not `2` now to give each failure a unique exit status? Probably doesn't matter too much - I'm not sure that we use the exit statuses anywhere

Reply:
Oh yes, good spot. I hadn't noticed the return codes, and don't think they're used. Nonetheless, I've changed it to a `1 2 3` sequence for now.